api/routes/admins.js

```diff
@@ -0,0 +1,39 @@
+var express = require('express');
+var router = express.Router();
+
+
+const Admin = require("../../db/models/Admin");
+const Organization = require("../../db/models/Organization");
+
+
+router.get("/", async (req, res, next) => {
+    const admins = await Admin.find();
+    return res.send(admins);
+});
+router.get("/login", async (req, res, next) => {
+    let email = req.body.email;
+    Admin.findOne({ email: email }).then((admin) => {
+        Organization.findOne({ _id: admin.organization }).then((organization) => {
+            return res.send(organization)
+        })
+    })
+});
+
+router.post("/", (req, res, next) => {
+    Organization.findOne({ _id: req.body.organization }).then(async (organization) => {
+        const admin = new Admin({
+            email: req.body.email,
+            password: req.body.password,
```

Comment:
ideally we wouldn't want to save a password as text. we could get hacked. we would want to use some sort of hash. for now, let's just add a comment saying we'll change this later.

Reply:
addressed